overlord/devicestate/systems.go

```diff
@@ -84,6 +84,10 @@ func systemFromSeed(label string, current *currentSystem) (*System, error) {
 	if err := s.LoadAssertions(nil, nil); err != nil {
 		return nil, fmt.Errorf("cannot load assertions: %v", err)
 	}
```

Comment:
this could move into the split helper, no? if we want to , we can improve the error message here, if the issue is that if we do our own LoadAssertions we get a better error

Reply:
I tweaked the error a tiny bit here to include the seed label